packages/patternfly-3/patternfly-react/package.json

```diff
@@ -50,8 +50,8 @@
   },
   "peerDependencies": {
     "prop-types": "^15.6.0",
-    "react": "^16.3.2",
-    "react-dom": "^15.6.2 || ^16.2.0"
```

Comment:
I'm don't believe we should change the dependencies for PatternFly 3. There may have been a reason for supporting these specific versions?